docs/usage.rst

```diff
@@ -247,6 +247,22 @@ xrootd`` command-line option to use that protocol instead of HTTP/HTTPS:
       -> File: ./5500/mass4l_combine.png
     ==> Success!
 
+**Downloading engine**
```

Comment:
```
*Select downloader engine*
```

to go better with the other subtitles that use the verb style such as "Filter by name..."